plugins/permanentpen.md

```diff
@@ -17,7 +17,7 @@ The Permanent Pen can also be used to create user notes and other useful comment
 
 The default properties of Permanent Pen are:
 
-```js
+```
```

Comment:
Was there a reason the JS markup hint was removed from here? As this is technically a JS snippet.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d